src/core/mesh/qgsmesheditor.cpp

```diff
@@ -257,8 +257,6 @@ void QgsMeshEditor::applyEditOnTriangularMesh( QgsMeshEditor::Edit &edit, const
 void QgsMeshEditor::applyAdvancedEdit( QgsMeshEditor::Edit &edit, QgsMeshAdvancedEditing *editing )
 {
   applyEditOnTriangularMesh( edit, editing->apply( this ) );
-
-  emit meshEdited();
```

Comment:
why this is not required anymore?

Reply:
it was double emitted, already emitted by the undo stack of the mesh layer. Removed here because, the second emit remove the element selection.